addons/crm_addons/web_outlook/src/taskpane/api.js

```diff
@@ -0,0 +1,14 @@
+const api = {
+    baseURL: localStorage.getItem('baseURL') || "https://9d80deba.ngrok.io",
+    createLead: "/crm/addons/lead/create",
+    getLeads: "/crm/addons/lead/get_by_partner_id",
+    deleteLead: "/crm/addons/lead/delete",
+    getPartner: "/crm/addons/partner/get",
+    logMail: "/crm/addons/log_single_mail_content",
+    token: "/crm/addons/token",
```

Comment:
detail: you should use different variable name than `token` for an endpoint, not easy to grep nor understand the purpose